src/pudl/transform/eia923.py

```diff
@@ -1640,3 +1640,65 @@ def _core_eia923__yearly_byproduct_expenses_and_revenues(
     df.columns = df.columns.str.replace("_1000_dollars", "")
 
     return df
+
+
+@asset(io_manager_key="pudl_io_manager")
+def _core_eia923__emissions_control(
```

Comment:
This should be renamed to include the temporal resolution, which I think is yearly, so `_core_eia923__yearly_emissions_control`, and the resource definition will need to be updated as well.